.github/workflows/build-pgazure-nightlies.yml

```diff
@@ -25,47 +28,46 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          # We will remove el/8 support for all apps in the future. For this project packaging image should be updated.But we
-          # don't want to invest.
-          # - el/8
-          # temporarily removed from nightlies since pg15 beta is not being downloaded for ol/7
-          # - ol/7
+          - el/7
+          - debian/buster
+          - debian/bullseye
+          - ubuntu/bionic
           - ubuntu/focal
+          - ubuntu/jammy
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
 
       # This step is to fetch the images unanonymously to have higher bandwidth
       - name: Login to Docker Hub
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USER_NAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
       - name: Clone tools branch
-        run: git clone -b pgaz_test --depth=1  https://github.com/citusdata/tools.git tools
-
-      - name: Clone build branch
-        run: git clone -b "all-pg-azure-storage" --depth=1  https://github.com/citusdata/packaging.git packaging
+        run: git clone -b v0.8.25 --depth=1  https://github.com/citusdata/tools.git tools
 
       - name: Install package dependencies
-        run: sudo apt-get update && sudo apt-get install -y libcurl4
+        run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Install python requirements
         run: python -m pip install -r tools/packaging_automation/requirements.txt
 
       - name: Build packages
         run: |
+          echo "INSTALL_RUST=${INSTALL_RUST}"
+          echo "CI=${CI}"
           python -m  tools.packaging_automation.citus_package \
           --gh_token "${GH_TOKEN}" \
           --platform "${{ matrix.platform }}" \
           --build_type "nightly" \
           --secret_key "${PACKAGING_SECRET_KEY}" \
           --passphrase "${PACKAGING_PASSPHRASE}" \
           --output_dir "$(pwd)/packages/" \
-          --input_files_dir "$(pwd)/packaging" \
-          --package_unsigned
+          --input_files_dir "$(pwd)"
```

Comment:
Normally nightlies execute on develop branch and in that branch I clone packaging repo and change branch
I had copied this pipeline from develop branch. However, I changed the logic and used the existing codebase since it already includes pgazure configurations